exercises/high-scores/.meta/solutions/high_scores.rb

```diff
@@ -0,0 +1,31 @@
+class HighScores
+  def initialize(scores)
+    @scores = scores
+  end
+
+  def scores
+    @scores
+  end
+
+  def add(new_score)
+    scores << new_score
+  end
+
+  def highest
+    scores.max
+  end
+
+  def latest
+    scores.last
+  end
+
+  def top_3
```

Comment:
I'm not sure what's wrong with the name. On second thought I don't like `podium`, because this is more like a one person's list. How about: `3_personal_bests`? hmmm. Or wait: top_3? ;-)

Reply:
I think we can leave it as is for now and circle back later if needed.